flow/genericactors.actor.h

```diff
@@ -1209,6 +1211,14 @@ Future<T> brokenPromiseToMaybeDelivered(Future<T> in) {
 	}
 }
 
+ACTOR template <class T, class U>
+void tagAndForward(Promise<T>* pOutputPromise, U value, Future<Void> signal, double startTime, Reference<Histogram> latencyHistogram) {
+	state Promise<T> out(std::move(*pOutputPromise));
+	wait(signal);
+	out.send(std::move(value));
+	latencyHistogram->sampleSeconds(timer_monotonic() - startTime);
```

Comment:
should this be moved before `out.send()`? `out.send()` will invoke callbacks inline.

Reply:
Thanks for the catch! Fixed.